governance-aepp/src/data/help.json

```diff
@@ -1,7 +1,7 @@
 {
   "home": {
-    "headline": "The Governance Aepp",
-    "text": "On the home page all listed polls are shown. The different tabs can be used to filter and/or sort the polls. To navigate the aepp, use the buttons at the bottom of the screen.",
+    "headline": "The Voting Aepp",
```

Comment:
why change the title? where has this been decided?

Reply:
In the bridge